lti/lti-common/src/java/org/sakaiproject/lti13/LineItemUtil.java

```diff
@@ -120,13 +120,16 @@ public static String constructExternalId(Map<String, Object> content, SakaiLineI
 	 */
 	// TODO: Remember to dream that someday this will be JSON :)
 	public static String constructExternalId(Long tool_id, Map<String, Object> content, SakaiLineItem lineItem)
+	{
+		Long content_id = (content == null) ? null : ((Integer) content.get(LTIService.LTI_ID)).longValue();
```

Comment:
I think this is not going to work on Oracle...

Reply:
@jesusmmp Could you expand on what would/might break for Oracle?

Reply:
The casting to Integer:

https://github.com/sakaiproject/sakai/pull/13661/files#diff-17263379da42a2007b1b53f17db8aea84add508a5c71d29c2a318d985ad70500R748

Reply:
Ok, thanks. I'll try the cited alternative in testing. If I recall correctly, I originally set the cast to Long but encountered a runtime exception when testing against mariadb.